components/AddToCart.js

```diff
@@ -0,0 +1,71 @@
+import cookie from 'js-cookie';
+import React from 'react';
+
+export default function AddToCart(props) {
+  const itemsInCart = cookie.getJSON('cart') || [];
+  console.log('jonas', itemsInCart);
+
+  // useEffect(() => {
+  //   cookie.set('cart', itemsInCart);
+  //   console.log(itemsInCart);
+  // }, [itemsInCart]);
+
+  function MakeCookies() {
+    const product = {
+      name: props.product.name,
+      price: props.total,
+      id: props.product.id,
+      amount: props.pieces,
+      img: props.product.img,
+      info: props.product.info,
+    };
+
+    const itemFilter = itemsInCart.find((item) => item.id === product.id);
```

Comment:
Maybe a better name would be `item`? If it gives you an error in the `find` function, you could call the inner one `currentItem` or `i`...